api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -453,6 +453,10 @@
               "a workflow should have been created")
           (finally
             (endpoints/stop-workload workload)))
+        ;; Note: when the workload's workflows have finished,
+        ;; we expect a notification for each workflow
+        ;; to be emitted to the Slack channels in
+        ;; wfl.tools.workloads/watchers.
```

Comment:
Does this comment substitute for verifying the notifications arrived?

Reply:
Intended more of a heads up, but you're right that a verification would be more appropriate… unfortunately I'm not sure how to do that, the pending Slack test indicates that its attempt at verification is flaky when run in GitHub Actions:

https://broadinstitute.atlassian.net/browse/GH-1553